### DISCUSSION TITLE
How to make my application for Student Developer Pack?

### DISCUSSION THREAD

**Discussion by Shin-suechtig** [community/community#36536] [GitHub Education]
Hi. I'm a Japanese senior high school student, and wanna get the Student Developer Pack. For several times, I filled in the blank with my school name, the reason why I wanna get that, a picture of my dated school ID card and translation for what's written on the card into English, But every time it was rejected at the *pre-check* stage with the following error message. Huh, I think I've met the conditions they say. When sending a request, I had set my display name as my real full-name and included my school name on my bio -- just when sending it, even unset now -- , and been exactly at my school, never far away from my school (Or does the last message see as a problem that I live a little bit far away from my school? If so, why'd it be a problem?). 

Error message:

Your application cannot be reviewed until you fix the following:

- You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.
- Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?
- You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

**Comment by HalemoGPA** [comment]
I think you should be in a college not a high school.
